TFT/src/User/API/Language/language_keywords.h

```diff
@@ -352,6 +352,9 @@ extern "C" {
 #define LANG_KEY_CLICK_FOR_MORE               "label_click_for_more:"
 #define LANG_KEY_EXT_TEMPLOW                  "label_ext_templow:"
 #define LANG_KEY_HEAT_HOTEND                  "label_heat_hotend:"
+#define LANG_KEY_DESIRED_TEMPLOW              "label_desired_templow:"
+#define LANG_KEY_WAIT_HEAT_UP                 "label_wait_heat_up:"
```

Comment:
use the already existing similar keywords: LANG_KEY_TUNE_EXT_DESIREDVAL

Reply:
Fixed